types/auth0/auth0-tests.ts

```diff
@@ -633,3 +633,10 @@ management.verifyCustomDomain({ id: 'cd_0000000000000001' }, (err, domainVerific
 
 management.deleteCustomDomain({ id: 'cd_0000000000000001' }).then(() => console.log('deleted'));
 management.deleteCustomDomain({ id: 'cd_0000000000000001' }, err => console.log('deleted'));
+
+// User enrollment
+management.getGuardianEnrollments({ id: 'cd_0000000000000001' }).then((enrollment) => console.log(enrollment))
+management.getGuardianEnrollments({ id: 'cd_0000000000000001' },(err, enrollment) => console.log(enrollment))
```

Comment:
```suggestion
management.getGuardianEnrollments({ id: 'cd_0000000000000001' }).then((enrollments) => console.log(enrollments))
management.getGuardianEnrollments({ id: 'cd_0000000000000001' },(err, enrollments) => console.log(enrollments))
```
so this a list, shoudl ends with `s`, to point what's being returned. Am I correct?

Reply:
Updated

Yes, it should be in plural to make it clearer